Genesis: Remove empty input

Geod24 opened the pull request:
This Input was never used, and was simplify superfluous.

Geod24 committed:
Genesis: Remove empty input

This Input was never used, and was simplify superfluous.

Geod24 commented on source/agora/utils/PrettyPrinter.d:
```diff
@@ -274,10 +274,13 @@ private struct TransactionFmt
+                    this.value.inputs.length,
+                    this.value.inputs.map!(v => InputFmt(v)).chunks(InputPerLine));
+            else
+                formattedWrite(sink, "Type : %s, Inputs: None\n", this.value.type);
```
This special case is so that we don't have trailing whitespace at the end of the line, which can be a PITA.

AndrejMitrovic commented:
@TrustHenry do you now see what Mathias meant?

TrustHenry reviewed (approved):
I can see it well.

Geod24 commented:
Well I'll rework the handling of blocks first...

TrustHenry commented:
https://github.com/bpfkorea/agora/blob/99af27cb0d133584e83018f8b44415579118411c/source/agora/test/Base.d#L1060
It applies here too, right?

Geod24 committed:
Make UnitTestGenesisTransactions private again

This was made public in PR877, but by accessing them through the Genesis block,
we can make it private again and we need not assume there's a single Genesis tx.

TrustHenry commented:
Is the rework still in progress?

Geod24 commented:
Yes. For the moment you can go with the `inputs.length == 1` in your PR and we'll fix it later.

MichaelKim20 commented:
@Geod24 If you change the genesis_block in the config file to the values of the links, the system integration test will be successful.
https://gist.github.com/MichaelKim20/edfc5d418ecc7f02857588b86312198c

MichaelKim20 commented:
The fourth node is not a validator.  However, I confirmed that the Genesis block must be the same as the other nodes.

Geod24 commented:
Ah I always forget we have 4 now.

Geod24 commented:
Thanks @MichaelKim20
